test/sqllogictest/explain/optimized_plan_as_text.slt

```diff
@@ -776,20 +776,20 @@ WHERE a = c and d = e and b = f
 ----
 Explained Query:
   Project (#0{a}, #1{b}, #0{a}, #3{d}, #3{d}, #1{b})
-    Filter (#0{a}) IS NOT NULL
-      Join on=(#0{a} = #2{c} AND #1{b} = #5{f} AND #3{d} = #4{e}) type=differential
+    Filter (#0{a}) IS NOT NULL AND (#1{b}) IS NOT NULL AND (#3{d}) IS NOT NULL
+      Join on=(#0{a} = #2{c} AND #1{b} = #5{f} AND #3{d} = #4{e}) type=delta
         ArrangeBy keys=[[#0{a}]]
-          ReadIndex on=t t_a_idx=[differential join]
-        ArrangeBy keys=[[#0{c}]]
-          ReadIndex on=u u_c_idx=[differential join]
-        ArrangeBy keys=[[#0{e}, #1{f}]]
-          Filter (#0{e}) IS NOT NULL AND (#1{f}) IS NOT NULL
-            ReadIndex on=v v_e_idx=[*** full scan ***]
+          ReadIndex on=t t_a_idx=[delta join 1st input (full scan)]
+        ArrangeBy keys=[[#0{c}], [#1{d}]]
+          ReadIndex on=u u_c_idx=[delta join lookup] u_d_idx=[delta join lookup]
+        ArrangeBy keys=[[#0{e}]]
+          ReadIndex on=v v_e_idx=[delta join lookup]
```

Comment:
Probably a good change: 1 less arrangement on inputs because of a reuse + 1 less intermediate arrangement due to switching from differential to delta join.